src/libs/suggest.js

```diff
@@ -63,11 +61,10 @@ export const getInputValue = item => {
 
 export const fetchSuggests = (query, options = {}) =>
   suggestResults(query, {
-    withCategories: options.withCategories || false,
-    useFocus: options.useFocus || SUGGEST_USE_FOCUS,
-    focusMinZoom: options.focusMinZoom || SUGGEST_FOCUS_MIN_ZOOM,
-    maxItems: options.maxItems || SUGGEST_MAX_ITEMS,
-    maxFavorites: options.maxFavorites || !query ? 5 : 2,
+    withCategories: options.withCategories ?? false,
```

Comment:
Is this change just a question of code style preference?

Reply:
Not exactly, `optional_value ?? default_value` should be preferred to `optional_value || defaut_value` here. It doesn't make a difference when the default is `false`. But for other params below, the default value is `true` and the optional value was always ignored:  
`false ?? true` evaluates to `false`  
whereas
`false || true` evaluates to `true`

Reply:
Ah yes of course! Sorry for the dumb question :D
I always try to give default false to boolean options for this kind of reason, but sometimes true is cool too.